cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -98,10 +124,9 @@ def record_type_is_deprecated(cls, v):
     @validator("oid_as_pk")
     @classmethod
     def oid_as_pk_is_deprecated(cls, v):
-        logger.warning(
-            "oid_as_pk is deprecated. Just supply an Id column declaration and it will be inferred."
+        raise ValueError(
+            "oid_as_pk is no longer supported. Include the Id field if desired."
         )
```

Comment:
Do you think there's any risk of still needing to run old metadeploy plans which use oid_as_pk?

Reply:
Good question. Very few MetaDeploy plans run data loads at all. NPSP's Trailhead installer does, but we shouldn't have a need to use an old version since that would not replicate a customer org. Grants Management does, but is very recent; foundationConnect's partner enablement installer does, but was never rolled out publicly.

@jstvz @jofsky Are you aware of any potential risk to older EDU installers that use data load tasks?

Reply:
I took a look as well and don't see any clear risks.